ADOPTERS.md

```diff
@@ -106,3 +106,4 @@ This list is sorted in the order that organizations were added to it.
 | [Evaneos](https://www.evaneos.com/) | [@grams](https://github.com/grams) | Crossplane has become Evaneos’ primary infrastructure-as-code solution for our latest SaaS environment and our developer self-service platform, using compositions and in-house open-source providers. It is also the IaC target for our Marketplace and legacy activities. |
 | [Ambev Tech](https://www.ambevtech.com.br/) | [@cencristian](https://github.com/cencristian), [@dimitridittrich](https://github.com/dimitridittrich), [@rafaelvilarinho](https://github.com/rafaelvilarinho) | We are using Crossplane as an executor in our IDP to provide infrastructure in our production environment. |
 | [Aurora Innovation](https://aurorainnovation.com) | [@O5ten](https://github.com/O5ten) | At Aurora Innovation we use Crossplane to power our cloud infrastructure. We use it together with Kustomize and ArgoCD to achieve true GitOps for our development and platform teams in production environments |
+| [Netclab](https://netclab.dev) | [@mbakalarski](https://github.com/mbakalarski) | [netclab-xp](https://github.com/mbakalarski/netclab-xp) - declarative, GitOps-driven lifecycle management of physical, virtual, and containerized routers using Crossplane. Provides layered, vendor-neutral abstractions (FabricIP, EVPN, Router) built on OpenConfig and RESTCONF, hiding device-specific complexity behind reusable XRDs and compositions. |
```

Comment:
this URL in the "organization" column seems to resolve to your github account (https://github.com/mbakalarski) - is that intended, or is there a better link to use for your company or organization?

Reply:
sounds good, let's merge! 💪